File: C#/POO-Herencia/POO-Herencia/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace POO_Herencia
{
	internal class Persona
	{
		public string nombre;
		protected int edad;
		private string cedula;
		const int salario = 60;
		public Persona(string nombre, int edad, string cedula)
		{
			this.nombre = nombre;
			this.edad = edad;
			this.cedula = cedula;
		}
		
		public string Nombre{ 
			get { return nombre; } 
			set { nombre = value; }
		}
		public int Edad
		{
			get { return edad; }
			set { edad = value; }
		}
		public string Cedula
		{
			get { return cedula; }
			set { cedula = value; }
		}
		public virtual string Hablar()
		{
			return "Persona: Hola mi nombre es: " + this.nombre;
		}
		public virtual string Calificar()
		{
			return "Persona: Persona calificada";
		}

		public override string ToString()
		{
			return "Nombre: " + this.nombre + " Edad: " + this.edad + " Cedula: " + this.cedula;
		
		}
	}
}

File: JAVA/PARCIAL II/POO-Herencia-Exposicion/src/poo/herencia/Persona.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poo.herencia;

/**
 *
 * @author Jeremy
 */
public class Persona {

    public String nombre;
    protected int edad;
    private String cedula;

    public Persona(String nombre, int edad, String cedula) {
        this.nombre = nombre;
        this.edad = edad;
        this.cedula = cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public String getCedula() {
        return cedula;
    }

    public void setCedula(String cedula) {
        this.cedula = cedula;
    }

    
    
    public String Hablar() {
        return "Persona: Hola, mi nombre es " + this.getNombre();
    }

    public String Calificar() {
        return "Persona: Calificando";
    }

    @Override
    public String toString() {
        return "Persona{" + "nombre=" + nombre + ", edad=" + edad + ", cedula=" + cedula + '}';
    }
     public int Suma(int x, int y){
        return x+y;
    }

}
